check_tv.py

```
@@ -36,10 +36,13 @@
 from requests.exceptions import RequestException
 import json
 import threading
-import cv2
 import getopt
 import re
 import shutil
+from twitchAPI.twitch import Twitch
+from twitchAPI.type import AuthScope
```

Ruff (0.8.2)

`twitchAPI.type.AuthScope` imported but unused

Remove unused import: `twitchAPI.type.AuthScope`

(F401)